overlord/snapshotstate/backend/restorestate.go

```diff
@@ -56,7 +56,7 @@ func (rs *RestoreState) Cleanup() {
 }
 
 func restoreStateFilename(fn string) string {
-	return fmt.Sprintf("%s.~%s~", fn, strutil.MakeRandomString(9))
+	return fmt.Sprintf("%s.~%s~", fn, randutil.MakeRandomString(9))
```

Comment:
WDYT about renaming that to `randutil.RandomString(..)` so that it's more in line naming-wise with `RandomDuration()`?

Reply:
that sounds good

Reply:
done